testing/regress/ecl/genjoin.ecl

```diff
@@ -25,8 +25,10 @@ jrec := RECORD,MAXLENGTH(100)
     UNSIGNED1 i;
     STRING3 lstr;
     STRING3 rstr;
-    UNSIGNED1 c;
```

Comment:
I need full coverage of various types of join and denormalize to make sure the match flags are being set correctly.